modelon/impact/client/experiment_definition.py

```diff
@@ -745,268 +730,26 @@ def to_dict(self):
                         "solverOptions": self._solver_options,
                         "simulationLogLevel": self._simulation_log_level,
                     },
+                    "expansion": {"algorithm": str(self._expansion)},
                 },
                 "extensions": [ext.to_dict() for ext in self._extensions],
             }
         }
+        if not isinstance(self._expansion, FullFactorial):
+            exp_dict["experiment"]["base"]["expansion"]["parameters"] = asdict(
+                self._expansion
+            )
```

Comment:
this is dependent on details of the expansion algorithm. It should be something like:
```
expansion_parameters = self._expansion.get_parameters_as_dict() # get_parameters_as_dict is abstract method
if expansion_parameters is not None:
  exp_dict["experiment"]["base"]["expansion"]["parameters"] = expansion_parameters
```